### ISSUE TITLE
Material 3 default font weight for titleLarge is incorrect

### ISSUE THREAD

**Issue by guyluz11** [flutter/flutter#122294]
## Steps to Reproduce

1. Create a Flutter app and set `themeData` to 
```dart
ThemeData(
      textTheme: GoogleFonts.poppinsTextTheme(),
      fontFamily: GoogleFonts.poppins().fontFamily,
      useMaterial3: true,
    );
``` 
2. Inspect titleLarge fontWeight from the theme using `print(Theme.of(context).textTheme.titleLarge!.fontWeight);` 

**Expected results:** 
value **500**

**Actual results:** 
value **400**

As written [here](https://api.flutter.dev/flutter/material/TextTheme-class.html) `titleLarge` supposes to be of size `Medium` and according to [fonts google Poppins](https://fonts.google.com/specimen/Poppins) medium should be 500. 

By looking at the output from the Figma tool of Material 3 with poppins and titleLarge we can also see that it is not bold enough in flutter.

<details>
<summary>Code sample</summary>

```dart
```

</details>

<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by huycozy** [comment]
Hi @guyluz11 
I'm not sure which Flutter version you are using since there is no output of `flutter doctor -v`. But when I try this on the latest stable and master channels, I can see it printed `FontWeight.w500` from the console log. So I think it's fixed on the latest Flutter channels. 
I would suggest you switch to the latest master channel and retry:

```
flutter channel master
flutter upgrade
```

<details>
<summary>Sample code</summary>

```dart
import 'package:flutter/material.dart';
import 'package:google_fonts/google_fonts.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    print(Theme.of(context).textTheme.titleLarge!.fontWeight);
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        textTheme: GoogleFonts.poppinsTextTheme(),
        fontFamily: GoogleFonts.poppins().fontFamily,
        useMaterial3: true,
      ),
      home: const Scaffold(
        body: Center(
          child: Text('Hello world'),
        ),
      ),
    );
  }
}

```
</details>

<details>
<summary>flutter doctor -v (stable and master)</summary>

```bash
[✓] Flutter (Channel stable, 3.7.7, on macOS 13.0.1 22A400 darwin-x64, locale en-EE)
    • Flutter version 3.7.7 on channel stable at /Users/huynq/Documents/GitHub/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2ad6cd72c0 (10 hours ago), 2023-03-08 09:41:59 -0800
    • Engine revision 1837b5be5f
    • Dart version 2.19.4
    • DevTools version 2.20.1

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-33, build-tools 31.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio
    • Android Studio at /Applications/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] IntelliJ IDEA Community Edition (version 2022.1.1)
    • IntelliJ at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/IDEA-C/ch-0/221.5591.52/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.76.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.60.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 13.0.1 22A400 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 110.0.5481.177

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

```bash
[!] Flutter (Channel master, 3.9.0-1.0.pre.144, on macOS 13.0.1 22A400 darwin-x64, locale en-EE)
    • Flutter version 3.9.0-1.0.pre.144 on channel master at /Users/huynq/Documents/GitHub/flutter_master
    ! Warning: `flutter` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/flutter, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/dart, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ac550c6356 (39 minutes ago), 2023-03-09 21:59:21 -0500
    • Engine revision e9ca7b2c45
    • Dart version 3.0.0 (build 3.0.0-313.0.dev)
    • DevTools version 2.22.2
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly to perform update checks and upgrades.

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-33, build-tools 31.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio
    • Android Studio at /Applications/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] IntelliJ IDEA Community Edition (version 2022.1.1)
    • IntelliJ at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/IDEA-C/ch-0/221.5591.52/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.76.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.60.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 13.0.1 22A400 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 110.0.5481.177

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 2 categories.
```
</details>

And I will close this issue as fixed. If you still experience this issue, kindly leave a comment below and I will reopen this. Thank you!

**Comment by guyluz11** [comment]
@huycozy thanks for the quick response.



<details>
  <summary>flutter doctor -v (stable and master)</summary>

```dart
[√] Flutter (Channel master, 3.9.0-1.0.pre.144, on Microsoft Windows [Version 10.0.22621.1265], locale en-IL)
    • Flutter version 3.9.0-1.0.pre.144 on channel master at C:\Users\guy\Programs\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ac550c6356 (4 hours ago), 2023-03-09 21:59:21 -0500
    • Engine revision e9ca7b2c45
    • Dart version 3.0.0 (build 3.0.0-313.0.dev)
    • DevTools version 2.22.2

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
    • Android SDK at C:\Users\guy\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.1
    • Java binary at: C:\Users\guy\Programs\Android\Android_Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-9505619)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.4.3)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.4.33205.214
    • Windows 10 SDK version 10.0.22000.0

[√] Android Studio (version 2022.1)
    • Android Studio at C:\Users\guy\Programs\Android\Android_Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-9505619)

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22621.1265]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 110.0.5481.178
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 110.0.1587.63

[√] Network resources
    • All expected network resources are available.

• No issues found!

```
</details>


Your code sample does show `FontWeight.w500` because you checked it before the `ThemeData` got set.
I have used your example and created a second print after `ThemeData` and got `FontWeight.w400`.

<details>
  <summary>Sample code</summary>

```dart
import 'package:flutter/material.dart';
import 'package:google_fonts/google_fonts.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    print(Theme.of(context).textTheme.titleLarge!.fontWeight);
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        textTheme: GoogleFonts.poppinsTextTheme(),
        fontFamily: GoogleFonts.poppins().fontFamily,
        useMaterial3: true,
      ),
      home: const Scaffold(
        body: BodyWidget(),
      ),
    );
  }
}

class BodyWidget extends StatelessWidget {
  const BodyWidget({super.key});

  @override
  Widget build(BuildContext context) {
    print(Theme.of(context).textTheme.titleLarge!.fontWeight);

    return const Center(
      child: Text('Hello world'),
    );
  }
}
```
</details>

**Comment by huycozy** [comment]
I reopen the issue for further discussion. Using your updated sample code above, I see the issue now. But it also happens even without using `poppins` font. Can you retry and confirm this?

**Comment by guyluz11** [comment]
Confirming that I can recreate it without any google fonts.

Wanted to note that for now I have solved it using hardcoded values in theme, here is an example for anyone else that has this issue

```dart
 textTheme: GoogleFonts.poppinsTextTheme().copyWith(
        bodySmall: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w400,
          ),
        ),
        bodyMedium: GoogleFonts.poppins(
            textStyle: GoogleFonts.poppins(
          fontWeight: FontWeight.w400,
        )),
        bodyLarge: GoogleFonts.poppins(
            textStyle: GoogleFonts.poppins(
          fontWeight: FontWeight.w400,
        )),
        labelSmall: GoogleFonts.poppins(
            textStyle: GoogleFonts.poppins(
          fontWeight: FontWeight.w500,
        )),
        labelMedium: GoogleFonts.poppins(
            textStyle: GoogleFonts.poppins(
          fontWeight: FontWeight.w500,
        )),
        labelLarge: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
        headlineSmall: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w400,
          ),
        ),
        headlineMedium: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w400,
          ),
        ),
        headlineLarge: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w400,
          ),
        ),
        titleSmall: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
        titleMedium: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
        titleLarge: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
        displaySmall: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
        displayMedium: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
        displayLarge: GoogleFonts.poppins(
          textStyle: GoogleFonts.poppins(
            fontWeight: FontWeight.w500,
          ),
        ),
      ),
```

**Comment by huycozy** [comment]
Thanks for confirming this and also providing the workaround. This issue is reproducible on the latest `stable` and `master` channels when using M3. It doesn't happen with M2.

<details>
<summary>Reproducible sample code</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    print('a: ${Theme.of(context).textTheme.titleLarge!.fontWeight}');
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        useMaterial3: true,
      ),
      home: Scaffold(
        body: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            Text('Hello world'),
            BodyWidget(),
          ],
        ),
      ),
    );
  }
}

class BodyWidget extends StatelessWidget {
  const BodyWidget({super.key});

  @override
  Widget build(BuildContext context) {
    print('b: ${Theme.of(context).textTheme.titleLarge!.fontWeight}');

    return const Center(
      child: Text('Hello world'),
    );
  }
}

```
</details>

<details>
<summary>flutter doctor -v (stable and master)</summary>

```bash
[✓] Flutter (Channel stable, 3.7.7, on macOS 13.0.1 22A400 darwin-x64, locale en-EE)
    • Flutter version 3.7.7 on channel stable at /Users/huynq/Documents/GitHub/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 2ad6cd72c0 (10 hours ago), 2023-03-08 09:41:59 -0800
    • Engine revision 1837b5be5f
    • Dart version 2.19.4
    • DevTools version 2.20.1

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-33, build-tools 31.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio
    • Android Studio at /Applications/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] IntelliJ IDEA Community Edition (version 2022.1.1)
    • IntelliJ at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/IDEA-C/ch-0/221.5591.52/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.76.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.60.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 13.0.1 22A400 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 110.0.5481.177

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

```bash
[!] Flutter (Channel master, 3.9.0-1.0.pre.144, on macOS 13.0.1 22A400 darwin-x64, locale en-EE)
    • Flutter version 3.9.0-1.0.pre.144 on channel master at /Users/huynq/Documents/GitHub/flutter_master
    ! Warning: `flutter` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/flutter, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /Users/huynq/Documents/GitHub/flutter/bin/dart, which is not inside your current Flutter SDK checkout at /Users/huynq/Documents/GitHub/flutter_master. Consider adding /Users/huynq/Documents/GitHub/flutter_master/bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ac550c6356 (39 minutes ago), 2023-03-09 21:59:21 -0500
    • Engine revision e9ca7b2c45
    • Dart version 3.0.0 (build 3.0.0-313.0.dev)
    • DevTools version 2.22.2
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git" directly to perform update checks and upgrades.

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-33, build-tools 31.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio
    • Android Studio at /Applications/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] IntelliJ IDEA Community Edition (version 2022.1.1)
    • IntelliJ at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/IDEA-C/ch-0/221.5591.52/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.76.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.60.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 13.0.1 22A400 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 110.0.5481.177

[✓] Network resources
    • All expected network resources are available.

! Doctor found issues in 2 categories.
```
</details>

**Comment by bleroux** [comment]
@guyluz11 Thanks for the detailed report!
@huycozy Thanks for pointing out that it is not related to a specific font and that it happens only with M3.

After investigation, the font weight for `titleLarge` in M3 is set to 400 here:

https://github.com/flutter/flutter/blob/bd4c792ba4ed945d97a11dd36bb9a520987b6659/packages/flutter/lib/src/material/typography.dart#L756-L772
This value comes from the M3 typography specification: https://m3.material.io/styles/typography/type-scale-tokens

![Capture d’écran du 2023-03-31 09-02-21](https://user-images.githubusercontent.com/840911/229047423-7d25dc7c-0fb4-4206-975d-a2da5bdbde25.png)

'Title' seems to be the only "role" which variants ('small', 'medium' and 'large') do not have the same font weight. On the other side, in Material 2 it was common to have such variations (for instance, titleLarge font Weight was 500, titleMedium 400 and titleSmall 500).

@HansMuller Is it possible to get confirmation from the MD team that this value is correct or if there is a typo in the spec?

**Comment by HansMuller** [comment]
@esouthren - can you verify that  font weight for titleLarge in M3 is supposed to be 400?

**Comment by guidezpl** [comment]
I can confirm this, closing this issue.

**Comment by guyluz11** [comment]
So if 400 is the correct value it should get changed in "Figma tool of Material 3" and in [google Poppins font](https://fonts.google.com/specimen/Poppins) spec page

**Comment by guidezpl** [comment]
For Google Fonts, could you please file feedback via the icon in the lower left? I'm not sure which Figma tool you're speaking of, but there should be a way to submit feedback. If not, let me know and I'll help get your feedback to the right folks.

**Comment by guyluz11** [comment]
I have opened for Google popins using the icon like you asked.

As described in [material 3 get started docs](https://m3.material.io/get-started) there is a link to Figma in the "Design" section
https://www.figma.com/community/file/1035203688168086460


Also, we have opened tickets for both places for them to check that they use the correct values around when the issue got opened and didn't get any response.
If there is another way we can communicate the issue it will be good.
